### PR TITLE
(chore) Add Team OAS documentation

### PR DESCRIPTION
**Pull request by filipecabaco**
Added Open API Specs for Team Management endpoint.

### CODE REVIEW

File: lib/logflare_web/open_api_schemas.ex [hunk 1]
```diff
@@ -76,11 +76,63 @@ defmodule LogflareWeb.OpenApiSchemas do
...
+        bigquery_dataset_location: %Schema{type: :string},
+        bigquery_dataset_id: %Schema{type: :string},
+        api_quota: %Schema{type: :integer},
+        company: %Schema{type: :string},
+        token: %Schema{type: :string}
+      },
+      required: [:name]
+    })
+
+    def response(), do: {"Team Response", "application/json", __MODULE__}
```
Comment (Ziinc) [inline lib/logflare_web/open_api_schemas.ex:103]: shouldn't this be "User Response"?

**Review by Ziinc — APPROVED** [review]
seems like a lot of boilerplate code, can consider writing a macro all the schema builderplate in the future.

File: lib/logflare_web/open_api_schemas.ex (hunk 1 above)
Comment (filipecabaco) [reply lib/logflare_web/open_api_schemas.ex:103]: yes it should! thank you

**Comment by filipecabaco** [comment]
I was thinking of doing that 👀 will tackle it when adding the next doc

**Pull request merged**